src/index.ts

```diff
@@ -68,11 +67,19 @@ app.get(`/:id`, (request, response) => {
 app.post(`/`, (request, response) => {
     // Check if it is a valid url
     const { url } = request.body;
-    if (!url || !config.urlRegex.test(url)) {
+    
+    try {
```

Comment:
I think something with `new URL(string).catch(() => {respond with HTTP 400});` should work here, but i'll try myself soonish.

Reply:
Do i look live i know ts? Man, thats directly from stackoverflow lmfao...